python/orca/src/bigdl/orca/automl/search/ray_tune/ray_tune_search_engine.py

```diff
@@ -407,17 +407,9 @@ def trial_dirname_creator(trial):
 def get_data_from_part_refs(part_refs):
     from bigdl.orca.data.utils import ray_partitions_get_data_label
 
-    partitions = ray.get(part_refs)
+    partitions = [ray.get(part_ref) for part_ref in part_refs]
```

Comment:
These two way seems to be equivalent.

Reply:
`part_refs` is a list of `partition_ref`s, where `partition_ref` is a list of `ObjectRef`s for shards. We cannot directly get the data from ray when the data is [[`object_ref`], [`object_ref`, `object_ref`]], therefore only the changed way should work.